src/modules/75-cf/strings/strings.en.yaml

```
@@ -244,28 +244,26 @@ featureFlags:
     noData: No metrics found in this time range.
     totalEvaluations: Total Evaluations
   flagPipeline:
-    deleteModalTitle: Remove this pipeline from this flag?
+    deleteModalTitle: Remove this Pipeline from this Flag?
     deleteModalText:
-      If you remove the pipeline from the flag, the automated steps in this pipeline won’t be triggered, for example, automated approval steps.
-      The pipeline will be saved and you can add it again later.
+      If you remove the Pipeline from the Flag, the automated steps in this Pipeline won’t be triggered, for example, automated approval steps.
+      The Pipeline will be saved and you can add it again later.
 
-      Are you sure you want to separate the pipeline?
+      Are you sure you want to separate the Pipeline?
     title: Flag Pipeline
     noDataMessage: Automate processes with Flag Pipelines
     noDataDescription: Flag Pipelines help to automate routine processes like approval requests, notifications, Jira ticket creation or update
-    noDataButtonText: Add Flag Pipeline
     drawerTitle: Flag Pipelines
-    drawerDescription:
-      Flag pipelines allow to automate routine actions and execute automatically after any change. Flag pipelines
-      have no CI or CD steps, instead they have only one flag change step. Flag, target and environment are all set
-      to runtime input
+    drawerDescription: 'You can only set Pipelines that:
+      <li> Have a Flag stage configured
+      <li> Don’t include a build or deploy stage'
```

Looks like you've got list items, but no surrounding list
```suggestion
    drawerDescription: |
      You can only set Pipelines that:
      <ul>
        <li>Have a Flag stage configured</li>
        <li>Don’t include a build or deploy stage</li>
      </ul>
```

(note the pipe in the first line just tells YAML that multi-line text is coming and don't try to evaluate as anything else

---

cool, looks neater and easier to read too

---

Oh, remember to capitalise _Build_ and _Deploy_ as Harness nouns

---

AH I considered it - feel like we need a dictionary of Harness official nouns